Maestro/subscriptions.json

```diff
@@ -853,6 +859,36 @@
         "GithubRepo": "<trigger-repo>",
         "BranchToMirror": "<trigger-branch>"
       }
+    },
+    // Orchestrated build final package publish
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/product/cli/master/build.semaphore",
```

Comment:
Since CLI is already doing parallel builds on master (without prodcon) is this going to cause problems?

Reply:
Hmm. There could be issues, but it depends on how those builds go.

If the orchestrated build's BuildId always stays the same or goes up, this trigger will only fire once per BuildId *change* because the semaphore's just BuildId, and it's the trigger.

If the BuildId changes back to an old BuildId due to a respin, CLI would produce new packages, but the other repos' packages would still be in the feed with the same old versions and would be repushed. Overwriting is turned off though, so the worst that will happen is the utility build failing. (Which isn't a problem.)